xbmc/interfaces/json-rpc/JSONServiceDescription.cpp

```diff
@@ -1424,7 +1424,7 @@ bool CJSONServiceDescription::prepareDescription(std::string &description, CVari
   }
 
   if (description.at(0) != '{')
-    description = StringUtils::Format("{{{:s}}}", description);
+    description = StringUtils::Format("{{{:s}}}", description.c_str());
```

Comment:
I think the comment is wrong. fmt (at least version 4?) should be able to handle pretty much everything that has an `operator<<`. You can test the line without the `c_str()`, it will work.